pages/base_page.py

```diff
@@ -173,10 +171,6 @@ def report_list(self):
             select = Select(report_dropdown)
             return [opt.text for opt in select.options]
 
-        @property
-        def is_exploitable_crash_report_present(self):
-            return self.is_element_present(*self._exploitable_crash_report_locator)
-
```

Comment:
I'm filing a new issue now to handle the the use case for privileged users to see that the exploitable crash report is present. It an important, but less concerning security issue. A bit lower priority.